### ISSUE TITLE
TIPS "When restarting container with a new image, all configuration will be lost"

### ISSUE THREAD

**Issue by paolo1983** [m33ts4k0z/silabs-multipan-docker#2]
Isn't it enough to mount a specific folder or file of the container as volume in the host to solve it?

**Comment by m33ts4k0z** [comment]
yes you can actually specify a `volumes:` parameter in the `docker-compose.yml ` with something like:
`- /root/usr/local/`
or 
`- /home/localuser/usr/local/`

And this should persisit the config. Haven't tried it though.
